Application/EdFi.Ods.AdminApi/appsettings.json

```diff
@@ -33,5 +33,22 @@
             "Microsoft.Hosting.Lifetime": "Information"
         }
     },
-    "AllowedHosts": "*"
+    "AllowedHosts": "*",
+
+    "IpRateLimiting": {
+        "EnableEndpointRateLimiting": true,
+        "StackBlockedRequests": false,
+        "RealIpHeader": "X-Real-IP",
+        "ClientIdHeader": "X-ClientId",
+        "HttpStatusCode": 429,
+        "IpWhitelist": [],
+        "EndpointWhitelist": [],
+        "GeneralRules": [
+            {
+                "Endpoint": "POST:/Connect/Register",
```

Comment:
we should have the "/connect/token" endpoint as well.

Reply:
Based on the standup meeting comments, new changes on Rate Limit implementation will be reviewed on Demo meeting with Stephen.